src/Cognito/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "socialiteproviders/cognito",
+    "description": "Allow oauth connections from AWS Cognito to Laravel Socialite. This allows laravel to log users in from your cognito user pool. Based on https://socialiteproviders.com/Laravel-Passport, the code closely follows this structure so its not too hard to change to a Laravel Passport identity provider.",
+    "keywords": [
+        "laravel",
+        "oauth",
+        "cognito",
+        "aws",
+        "socialite"
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "JamesSiebert",
+            "email": "js@media360.com.au"
+        }
+    ],
+    "require": {
+        "php": "^7.2 || ^8.0",
+        "ext-json": "*",
+        "socialiteproviders/manager": "~4.0"
+    },
+    "require-dev": {
+        "mockery/mockery": "~0.9.4",
```

Comment:
unused.

Check the `composer.json` files for the other providers.

Reply:
Removed, thanks for the feedback :)